Sources/WalletConnectRelay/RPC/Methods/Subscription.swift

```diff
@@ -1,12 +1,41 @@
 
 import Foundation
+import WalletConnectUtils
 
 struct Subscription: RelayRPC {
 
     struct Params: Codable {
         struct Contents: Codable {
+
             let topic: String
             let message: String
+            let publishedAt: Date
```

Comment:
do we need to rewrite encoder code? 

we can parse it as is, make it private and create computed property: 
```swift
private let publishedAt: UInt64
var timestamp: Date {
return Date(milliseconds: publishedAt)
}
```